src/engraving/libmscore/textedit.cpp

```diff
@@ -249,9 +249,14 @@ bool TextBase::isEditAllowed(EditData& ed) const
         return false;
     }
 
-    bool ctrlPressed  = ed.modifiers & Qt::ControlModifier;
     bool shiftPressed = ed.modifiers & Qt::ShiftModifier;
 
+    if (shiftPressed && ed.key == Qt::Key_F2) {
+        return false; // Toggle Special Characters dialog
```

Comment:
I think we can find a better solution
see https://github.com/musescore/MuseScore/pull/11923#issuecomment-1166914602

Reply:
Okey, let's leave it as it is for now. Many changes will be needed